### DISCUSSION TITLE
8월 4일 피드백 내용 정리

### DISCUSSION THREAD

**Discussion by Oleevia01** [CodeMedics/Harulog-backend#4] [General]
1-1 메인(피드)
- 피드와 통계 자료의 통일성이 없다(사용자에게 통계 제공하는 것 좋지 않다는 의견)
- 1시간마다 랜덤으로 뿌리는 피드의 기준이 필요함
- 피드 기분 카테고리로 피드가 보이도록

1-2. 메인(피드_댓글)
- 보고싶지 않은 댓글 숨김 기능 
- 댓글 공감 기능

2.사용자 피드백
- 항목으로 피드백 작성할 수 있도록(ex.피드, 일기, 댓글 등등)
- View on 기능 고려해보기(바로 피드백 할 수 있도록)

3. 관리자 모니터링(신고)
- 블러처리(문구: 관리자에 의해 블러처리된 게시물입니다.)
- 피드 블라인드 처리가 잦은 사람에 대한 제재가 필요
- 신고 상태 진행에 대한 관리(처리됨, 처리되지 않음)
- 오신고 신고자 관리 필요
- 임시 가리기 처리 기능(오신고 걸러내서 다시 게시 가능하도록)

----------------------------------------------------------

★서비스 컨셉
- 피드 통해서 감정 공유
- 관리자의 사이트 관리로 사이트 신뢰도 올리기

-----------------------------------------------------------

★주요 기능

1. 도메인 로직(자동화)
2. 관리자 모니터링(필터링 장치가 필요)
-관리자 모니터링 관련 : 타 사이트 회원관리 프로세스 참고하기
3. 데이터 관리 및 처리
-비즈니스 지표로 사용자들의 사용 패턴에 대한 파악 등이 필요 
-가공된 데이터, 시각화 필요
-사이트 운영에 대해서도 참고할 수 있음(ex. 비오고 우울한 수요일 우울한 게시글이 많을 것이라 예상되면 사이트 컬러를 밝게 바꿔주는 등의 경우)

**Comment by jeongmino1** [comment]
피드백 내용 참고가 많이 되었습니다.
